ci/publish.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then
+    pip install ghp_import --install-option="--prefix=$HOME/.local"
+    "${HOME}/.local/bin/ghp-import" -n reference/book
+    git push -qf "https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git" gh-pages
```

Comment:
Why do this instead of the [built-in travis support](https://rust-lang-nursery.github.io/mdBook/continuous-integration.html#deploying-your-book-to-github-pages)?